sleuth/sleuth/enrich_tls.py

```diff
@@ -89,9 +102,102 @@ def check_compliance(compliance_policy, scs):
                 return "yes" if scs_desc in compliance_data[compliance_policy] else "no"
 
     return "error loading file"
+    
+    
+def audit_certs_issuer(certs, trusted_ca_list):
+    # as implemented now, we will get the tls certs in order so that the root
```

Comment:
Doctrings are recommended tor Python just below the function by using quotes "This is a description of some function or class" or triple quotes """ hello """ for multi-line doctrings.